### PR TITLE
Fix Autoupdater for emoji-picker

### PR DESCRIPTION
**Pull request by hebbet**

### CODE REVIEW

File: ajax/libs/emoji-picker/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   ],
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/one-signal/emoji-picker.git",
+    "target": "git://github.com/onesignal/emoji-picker.git",
```
Comment (PeterDaveHello) [inline ajax/libs/emoji-picker/package.json]: I think this should be `git://github.com/OneSignal/emoji-picker` ?

Comment (sashberd) [reply ajax/libs/emoji-picker/package.json]: Agreed with @PeterDaveHello

**Comment by maruilian11** [comment]
@hebbet Would you please modify your commit as Peter said?

**Comment by hebbet** [comment]
changed.

**Pull request closed without merging**